### DISCUSSION TITLE
Using an oval base

### DISCUSSION THREAD

**Discussion by schadechr** [florianwns/z5omes#130] [Q&A]
Hi - I am building a zome with this wonderful tool as a starting point.
In the location, there is limited room for the circular footprint of the zome, but I have room for a reasonably tall one, so I was wondering: Is there a way to create a zome with the tool with an oval base instead of the circular one? (basically "squeezing" it horizontally.)

I am using plywood plates, so I am not using any of the connection methods, but the way.
Regards, Christian

**Comment by OrangeJuce82** [comment]
hey [schadechr](https://github.com/schadechr), i don't see how it's possible to make zome with an oval base. If you have pictures or something like this, to show me how develop this option. Thx a lot

**Comment by schadechr** [comment]
Hello Florian, I have given this some thought and attached a presentation. I am not a structural engineer or a software developer, but I interact with both professions.

Why do we want to build an oval zome in the first place? We have a rooftop terrace in our vacation home in Greece, where we want to place it, but there is limited room for a full-size zome, and it will be used as an outdoor sleeping location during the hot  summer months. We still want it to have the same height and max diameter in the longest direction, but flatten it to fit in to the limited space. That is also why I am building it in a way that is easy to assemble and disassemble.

I have used the Giant Grass product design as an example since it adds horizontal struts to its construction, but I am building something else. I want to construct a zome only from cutout plywood panels, without struts, but the principle is the same. That is why I am using your “origami” output as my construction drawing.

I am assuming that the distance between the joint rows is the same, so the struts will not be the same length, technically making it no longer a zome.

I have tried to show how “squeezing” a zome will result in repositioning of the joints, but I cannot implement the last step of the design in 3D.

The current model we are using is this one:
Z5omes | A Zome Maker (florianwns.github.io)<https://florianwns.github.io/z5omes/?q=eyJudW1fc3BpcmFscyI6MTIsInRoZXRhX2RlZyI6NDcuOTIsImFuZ2xlX3VuaXQiOiJkZWciLCJ1c2VfaHJfbW9kZSI6ZmFsc2UsImhlaWdodF9yYXRpbyI6MC42NjY2NjY3LCJraXRlX3JhdGlvIjoxLCJtZWFzdXJpbmdfdW5pdCI6ImNtIiwiem9tZV9oZWlnaHRfaW5fbW0iOjI0MDAsInRpbWJlcl93aWR0aF9pbl9tbSI6MSwidGltYmVyX3RoaWNrbmVzc19pbl9tbSI6MzMsInpfdmFuaXNoaW5nX3BvaW50X2luX21tIjo2MDAuMDAwMDksImFzc2VtYmx5X21ldGhvZCI6MCwiYXNzZW1ibHlfZGlyZWN0aW9uIjowLCJ4cGFuc2lvbl9kaXJlY3Rpb24iOjEsImFkZF9mb290aW5nIjoxLCJyb29mX292ZXJmbG93X2luX21tIjowLCJmYWNlX2RpdmlzaW9uc19ieV9jcm93bnNfc3RyIjoiMCwwLDAsMCwwLDAsMCwwIiwiYmluZHVfcmF0aW9zX3N0ciI6IjEsMSwxLDEsMSwxLDEsMSIsImxhbmd1YWdlIjoiZW4iLCJoZWxwX2lzX2Rpc3BsYXllZCI6ZmFsc2V9>

Current mockup and oval concept are attached


Best regards, Christian



Fra: Florian REY ***@***.***>
Dato: søndag, 14. juli 2024 kl. 18.13
Til: florianwns/z5omes ***@***.***>
Cc: Christian Triantafillou Schade ***@***.***>, Author ***@***.***>
Emne: Re: [florianwns/z5omes] Using an oval base (Discussion #130)

hey schadechr<https://github.com/schadechr>, i don't see how it's possible to make zome with an oval base. If you have pictures or something like this, to show me how develop this option. Thx a lot

**Comment by OrangeJuce82** [comment]
Thx for the reply and the explanation. 


> Current mockup and oval concept are attached

I don't see attachments.

**Comment by schadechr** [comment]
Hi sorry about that, probably because I mailed you through the GitHub relay.

I created some download links here:

https://www.dropbox.com/scl/fi/wy7sgwx8rp2nj8zpvpnqv/Oval-zome.pdf?rlkey=2msjapu9qirltq8yjcn3u65pr&st=5sqobowb&dl=0
https://www.dropbox.com/scl/fi/xl8l5nboeiqh4fzjzbzmi/IMG_6528.jpeg?rlkey=6hxwcoj3ehj6bu77xzl4fj291&st=z8yn4iu3&dl=0

Would you mind sharing an email address so I can mail you directly?

regards, Christian

Med venlig hilsen/best regards
Christian Triantafillou Schade

[Image]
Agile Leadership Consultant
Corporate Revolutionary X
https://corprevx.com/
***@***.***

+45 231190 67<tel:+4523119067>
Sortemosevej 215
2730 Herlev
Denmark

Corporate Revolutionary X is a part of the New Media Crew ApS group
VAT DK0035386262


Fra: Florian REY ***@***.***>
Dato: onsdag, 17. juli 2024 kl. 20.04
Til: florianwns/z5omes ***@***.***>
Cc: Christian Triantafillou Schade ***@***.***>, Author ***@***.***>
Emne: Re: [florianwns/z5omes] Using an oval base (Discussion #130)

Thx for the reply and the explanation.

Current mockup and oval concept are attached

I don't see attachments.
